src/components/InstructorStatsCard/components/InstructorStat/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import {map} from 'lodash'
+import Anchor from 'components/Anchor'
+import Image from 'components/Image'
+
+export default ({count, label, graphics}) => (
+  <div>
+    <div className='flex items-center justify-center mb3'>
+      <div className='base f3 b mr2'>
+        {count}
+      </div>
+      <div>
+        {label}
+      </div>
+    </div>
+    <div className='flex flex-wrap justify-center'>
+      {map(graphics, graphic => (
+        <Anchor 
+          key={graphic.name}
+          url={graphic.http_url}
+        >
+          <Image 
+            src={graphic.logo_http_url}
+            alt={graphic.label}
+            className='w2 mr3 mb3 grow-large'
+          />
+        </Anchor>
+      ))}
+    </div>
+  </div>
+)
```

Comment:
This looks like this:
![image](https://cloud.githubusercontent.com/assets/5497885/24682198/a5140936-1955-11e7-9ba2-160e9e362f36.png)

Icons grow to indicate they can be tapped on hover/active/focus